compiler/damlc/daml-ide/src/DA/Daml/LanguageServer.hs

```diff
@@ -68,6 +77,39 @@ setIgnoreOptionalHandlers = PartialHandlers $ \WithMessage{..} x -> return x
     -- $/setTraceNotification which we want to ignore.
     where optionalPrefix = "$/"
 
+filesFromExecParams :: List Aeson.Value -> [NormalizedFilePath]
+filesFromExecParams (List files) = map (toNormalizedFilePath . T.unpack) (concatMap fileStringToPath files)
+            where fileStringToPath :: Aeson.Value -> [T.Text]
```

Comment:
The name `fileStringToPath` is very misleading. Maybe something like `collectTexts` is better.

Reply:
Simplified further we will not need that function